src/components/BaseStyles/BaseStyles.tsx

```diff
@@ -13,12 +13,12 @@
  * limitations under the License.
  */
 
-import React from 'react';
+import React, { FunctionComponent } from 'react';
 import { Global } from '@emotion/core';
 import { Theme } from '@sumup/design-tokens';
 
 import { createBaseStyles } from './BaseStylesService';
 
-export const BaseStyles = () => (
+export const BaseStyles: FunctionComponent = () => (
```

Comment:
The `FunctionComponent` type isn't accurate here, because it includes the `children` prop which isn't supported by this component.

Reply:
Oh nice catch. Never actually considered that. Thanks.